managed/resource_cluster_db_audit_export_config.go

```diff
@@ -0,0 +1,394 @@
+}
+
+func (m DefaultTrueModifier) Modify(ctx context.Context, req tfsdk.ModifyAttributePlanRequest, resp *tfsdk.ModifyAttributePlanResponse) {
+	// Check if the attribute is already set in the plan.
+	if !req.AttributePlan.IsNull() && !req.AttributePlan.IsUnknown() {
+		return // The value is already set, no modification needed.
+	}
+
+	// Set the default value to true.
+	resp.AttributePlan = types.Bool{Value: true}
+}
+
+func (r resourceClusterDbAuditExportConfigType) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{
+		Description: `The resource to manage DB Audit log configuration for a cluster in YugabyteDB Managed.`,
+		Attributes: map[string]tfsdk.Attribute{
+			"account_id": {
+				Description: "ID of the account this DB Audit log configuration belongs to.",
+				Type:        types.StringType,
+				Computed:    true,
+			},
+			"project_id": {
+				Description: "ID of the project this DB Audit log configuration belongs to.",
+				Type:        types.StringType,
+				Computed:    true,
+			},
+			"cluster_id": {
+				Description: "ID of the cluster with which this DB Audit log config will be associated ",
+				Type:        types.StringType,
+				Required:    true,
+			},
+			"exporter_id": {
+				Description: "ID of the exporter to which the DB Audit logs will be exported",
+				Type:        types.StringType,
+				Required:    true,
+			},
+			"config_id": {
+				Description: "ID of the DB Audit log configuration",
+				Type:        types.StringType,
+				Computed:    true,
+			},
+			"state": {
+				Description: "The stutus of association of cluster with DB Audit log config",
+				Type:        types.StringType,
+				Computed:    true,
+			},
+			"ysql_config": {
+				Description: "The specification for a DB Audit ysql export configuration",
+				Required:    true,
+				Attributes: tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+					"statement_classes": {
+						Description: "List of ysql statements",
+						Type:        types.SetType{ElemType: types.StringType},
+						Required:    true,
+					},
+					"log_settings": {
+						Description: "Db Audit Ysql Log Settings",
+						Optional:    true,
+						Attributes: tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+							"log_catalog": {
+								Description: "These system catalog tables record system (as opposed to user) activity, such as metadata lookups and from third-party tools performing lookups",
+								Type:        types.BoolType,
+								Optional:    true,
+								PlanModifiers: []tfsdk.AttributePlanModifier{
+									DefaultTrueModifier{}, // Apply your custom plan modifier here.
+								},
+								Computed: true,
+							},
+							"log_client": {
+								Description: "Enable this option to echo log messages directly to clients such as ysqlsh and psql",
+								Type:        types.BoolType,
+								Optional:    true,
+								PlanModifiers: []tfsdk.AttributePlanModifier{
```

Comment:
Why do you need a planModififer if you are not doing anything.

Reply:
Plan modifier is used to set the default value for various attributes under log settings. 
When the user does not provide say `log_client` we must defaut it to true. To maintain the initial and final state we use plan modifiers